client/src/Questions/Question.js

```diff
@@ -0,0 +1,23 @@
+import { Route, Switch, useRouteMatch } from "react-router-dom";
+import QuestionsForm from "./QuestionForm";
+import QuestionList from "./QuestionsList";
+
+const Questions = () => {
+  const { path } = useRouteMatch();
+
+  return (
+    <Switch>
+      <Route exact path={path}>
+        <QuestionsList />
+      </Route>
+      <Route path={`${path}/new`}>
+        <QuestionForm />
+      </Route>
+      <Route exact path={`${path}/:id/edit`}>
+        <QuestionList />
```

Comment:
In order to EDIT a question, it should display the QuestionForm component again, NOT the QuestionsList